scripts/astrokat-uvcoverage.py

```diff
@@ -31,13 +34,15 @@ def __init__(self,
         self.wavelength = phys.c / self.frequency
 
         self.ref_position = None
-        arr_desc = {'names': ('name', 'north', 'east', 'up', 'location'),
+        self.observer = None
+        arr_desc = {'names': ('name', 'east', 'north', 'up', 'location'),
                     'formats': ('S5', float, float, float, EarthLocation)}
```

Comment:
Does S5 mean up to 5 chars?  I'm wondering about SKA / MKAT+ dishes, which will have names like `s0013`, in which case 5 is enough.

Reply:
Yes, s5 means max of 5 characters are read, so if you give more than 5 it truncates.
I used the meerkat array naming convention -- 5 should be sufficient.